oe_corporate_site_info.post_update.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate Site Information post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Install Multi-value form element contrib module.
+ */
+function oe_corporate_site_info_post_update_install_multivalue_form_element_module(): void {
```

Comment:
We talked with @ademarco and decided to use numbers in OpenEuropa components (if not published also to d.o).